ui/pages/routes/routes.component.js

```diff
@@ -367,6 +372,9 @@ export default class Routes extends Component {
           }
         }}
       >
+        {isUnlocked && !isNetworkUsed && hasAnAccountWithNoFundsOnNetwork && (
```

Comment:
Nit: Could we assign this set of conditions to a variable above the JSX?  Like:

```
const shouldShowNetworkInfo = isUnlocked && !isNetworkUsed && hasAnAccountWithNoFundsOnNetwork;
```

... and then:

```
{shouldShowNetworkInfo && <NewNetworkInfo />}
```